tests/gocase/unit/type/zset/zset_test.go

```diff
@@ -1309,6 +1309,9 @@ func basicTests(t *testing.T, rdb *redis.Client, ctx context.Context, enabledRES
 		res := rdb.ZMScore(ctx, "zset", "a", "b").Val()
 		require.Equal(t, int64(1), int64(res[0]))
 		require.Equal(t, int64(2), int64(res[1]))
+
+		require.Equal(t, redis.Nil, int64(rdb.ZMScore(ctx, "non-existent-zset", "a", "b").Val()[0]))
+		require.Equal(t, redis.Nil, int64(rdb.ZMScore(ctx, "non-existent-zset", "a", "b").Val()[1]))
```

Comment:
It's obviously wrong. You can try to run golang test cases in your local and you can get some error messages.

Reply:
sorry, I will fix it

Reply:
1. do not cast it to `int64`, `int64` != `nil`;
2. `rdb.ZMScore(...)` can be reused.